pyproject.toml

```diff
@@ -14,7 +14,7 @@ dependencies = [
     "python-dotenv==1.0.1",
     "sqlalchemy==2.0.27",
     # ML/DL Base
-    "numpy>=1.26.0",
+    "numpy==1.26.4",
```

Comment:
Note: This was required as a downstream requirement by the ROCm pytorch wheels. I am looking into seeing if this is necessary.